x-pack/plugins/ingest_pipelines/public/application/components/pipeline_form/pipeline_form.tsx

```diff
@@ -27,6 +27,7 @@ export interface PipelineFormProps {
   saveError: any;
   defaultValue?: Pipeline;
   isEditing?: boolean;
+  disallowNameEdition?: boolean;
```

Comment:
I think you might mean `disallowNameModification` here, or maybe we could have something like `canEditName`

Reply:
Yes `canEditName` is a lot simpler, I will make the change